CHANGELOG

```diff
@@ -1,3 +1,7 @@
+v0.7.1
+ - Make Elasticity::Strategies::AliasIndex the default
+ - Use mapping instead of mappings, we wanna be consistent to ES not to elasticsearch-ruby
+ - Better automatic index name and document type
```

Comment:
We also need to add a note to the README about upgrading from one strategy to another, saying that they need to explicit define the old strategy in order to keep their indexes working.